facebook/delphiFacebook/R/variables.R

```diff
@@ -157,7 +153,7 @@ code_mental_health <- function(input_data) {
   wave <- unique(input_data$wave)
   assert(length(wave) == 1, "can only code one wave at a time")
 
-  if (wave >= 4) {
+  if (wave >= 4 & wave < 10) {
```

Comment:
```suggestion
  if (wave >= 4 && wave < 10) {
```
doesn't matter here, but `&&` is short-circuiting logical AND; `&` is vectorized AND